### ISSUE TITLE
*Prevent* release creation?

### ISSUE THREAD

**Issue by balthisar** [svenstaro/upload-release-action#54]
tl;dr: is there a technique for preventing the creation of a release if it doesn't already exist? I'd rather have it fail.

Longer version:

I'm actually using `workflow_dispatch` with a user-supplied ref to run my action, because it can produce useful artifacts for _any_ ref the user supplies, for example, an issue-specific debug branch, older versions, etc.

On the other hand, if the user-supplied ref is a tag that's associated with a release is provided, it would be awesome to upload the artifacts to the release automatically, but I don't want to create releases if they don't already exists.

Is there any solution to this that exists, even if it requires previous steps? Alternatively, if I were to supply a PR with a new attribute governing creating (or not) a new release, are you open to that?

Thanks!

**Comment by Porges** [comment]
I’d also like this option.
